src/containers/item-details/index.tsx

```diff
@@ -32,40 +30,10 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
             style={{ height: '100vh', position: 'sticky' }}
           >
             <Sticky>
-              {/* <div className="row">
-                <div className="col-2 flex-column d-flex justify-content-between">
-                  <button
-                    onClick={() => handleImageChange(item.image.url)}
-                    onKeyDown={() => handleImageChange(item.image.url)}
-                    className={`${selectedImg === item.image.url ? 'border' : ''}`}
-                    type="button"
-                  >
-                    <GalleryTab imageUrl={item.image.url} />
-                  </button>
-                  <button
-                    onClick={() => handleImageChange(item.image.url)}
-                    onKeyDown={() => handleImageChange(item.image.url)}
-                    className={`${selectedImg === item.image.url ? 'border' : ''}`}
-                    type="button"
-                  >
-                    <GalleryTab imageUrl={item.image.url} />
-                  </button>
-                  <button
-                    onClick={() => handleImageChange(item.image.url)}
-                    onKeyDown={() => handleImageChange(item.image.url)}
-                    className={`${selectedImg === item.image.url ? 'border' : ''}`}
-                    type="button"
-                  >
-                    <GalleryTab imageUrl={item.image.url} />
-                  </button>
-                </div>
-                <div className="col-10">
-                  <GalleryTab imageUrl={selectedImg} />
-                </div>
-              </div> */}
-              <GalleryTab imageUrl={selectedImg} NFTName={item.name} />
+              <GalleryTab imageUrl={item.image.url} NFTName={item.name} />
```

Comment:
How come you are including changes that don't correspond to the gallery filter tasks? Please do not mix things up